source/client/index.js

```diff
@@ -26,10 +27,13 @@ const store = createStoreWithMiddleware(reducer);
 // reduxRouterMiddleware.listenForReplays(store)
 
 ReactDOM.render(
-  <Provider store={store}>
+  <Provider store={ store }>
     <Router history={ browserHistory }>
-      <Route path="/" component={ createApp(React) } />
-      <Route path="/test-data" component={ createTestData(React) } />
+      <Route path='/' component={ createContainer(React) }>
+        <IndexRoute component={ createHome(React) } />
+        <Redirect from='/home' to='/' />
```

Comment:
Actually the Container contains a Navbar + Child. That way, on the IndexRoute "/", it renders the Home child.